include/contractor/contract_excludable_graph.hpp

```diff
@@ -91,7 +91,7 @@ inline auto contractExcludableGraph(ContractorGraph contractor_graph_,
         edge_container.Merge(toEdges<QueryEdge>(std::move(filtered_core_graph)));
     }
 
-    return GraphAndFilter{QueryGraph{num_nodes, std::move(edge_container.edges)},
+    return GraphAndFilter{QueryGraph{num_nodes, edge_container.edges},
```

Comment:
Here we could probably change the signature of constructor to accept rvalues, but I decided to just keep things "as is" for now.